### DISCUSSION TITLE
Deny users from seeing certain parts of their server

### DISCUSSION THREAD

**Discussion by MrFoxyOfficial** [pterodactyl/panel#4494] [Support] [answered]
I'm planning to use Ptero to host a Whitelabel bot, but I still want them to have access, so in case there's an issue of not being down or the bot not responding, they can restart it. 

Now, my bot src is not public and therefore, would love to block access to files, so that they can't take it and use it as their own. 

Is it possible to deny users from accessing the files in their server dashboard?

**Comment by trenutoo** [comment, marked as answer]
The owner of a server has access to all features of the panel. Sub-users have fine-grained permissions.
